tests/test_tftools.py

```diff
@@ -94,3 +95,20 @@ def test_batch_norm_3d():
     x = tf.placeholder(tf.float32, input_shape)
     with pytest.raises(ValueError):
         y = layers.batch_norm(x, '4d')
+
+
+def test_resize_image_like():
+    batch = 1
+    width_height = 5
+    width_height2 = 15
+    channel = 3
+    x = tf.placeholder(tf.float32, [batch, width_height, width_height, channel])
+    s = tf.placeholder(tf.float32, [batch, width_height2, width_height2, channel])
+    x_rshape = images.resize_images_like(x, s)
```

Comment:
In my opinion, test code should have names like this.  x and s are just simple variable placeholders that don't actually mean anything.  You are right that x_rshape should probably be x_resize